keps/sig-contributor-experience/0034-artifact-management.md

```diff
@@ -0,0 +1,124 @@
+
+The `Metadata` section above is intended to support the creation of tooling around the KEP process.
+This will be a YAML section that is fenced as a code block.
+See the KEP process for details on each of these items.
+
+## Table of Contents
+
+A table of contents is helpful for quickly jumping to sections of a KEP and for highlighting any additional information provided beyond the standard KEP template.
+[Tools for generating][] a table of contents from markdown are available.
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Stories [optional]](#user-stories-optional)
+      * [Story 1](#story-1)
+      * [Story 2](#story-2)
+    * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+    * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Drawbacks [optional]](#drawbacks-optional)
+* [Alternatives [optional]](#alternatives-optional)
+
+[Tools for generating]: https://github.com/ekalinin/github-markdown-toc
+
+## Summary
+This document describes how official artifacts (Container Images, Binaries) for the Kubernetes
+project are managed and distributed.
+
+
+## Motivation
+
+The motivation for this KEP is to describe a process by which artifacts (container images, binaries)
+can be distributed by the community. Currently the process by which images is both ad-hoc in nature
+and limited to an arbitrary set of people who have the keys to the relevant repositories. Standardize
+access will ensure that people around the world have access to the same artifacts by the same names
+and that anyone in the project is capable (if given the right authority) to distribute images.
+
+### Goals
+
+The goals of this process are to enable:
+  * Anyone in the community (with the right permissions) to manage the distribution of Kubernetes images and binaries.
+  * Fast, cost-efficient access to artifacts around the world through appropriate mirrors and distribution
+
+This KEP will have succeeded when artifacts are all managed in the same manner and anyone in the community
+(with the right permissions) can manage these artifacts.
+
+### Non-Goals
+
+The actual process and tooling for promoting images, building packages or otherwise assembling artifacts
+is beyond the scope of this KEP. This KEP deals with the infrastructure for serving these things via
+HTTP as well as a generic description of how promotion will be accomplished.
+
+## Proposal
+
+The top level design will be to set up a global redirector HTTP service (`artifacts.k8s.io`) 
+which knows how to serve HTTP and redirect requests to an appropriate mirror. This redirector
+will serve both binary and container image downloads. For container images, the HTTP redirector
```

Comment:
do we really want to serve binaries and container images in the same address?
will be there a prefix to indicate type of file?

Reply:
Why does it matter? The URL will know what kind of thing it is.